slowfs.py

```diff
@@ -15,14 +15,19 @@
 import sys
 import threading
 import time
-
-import fuse
-
+import fuse      # Filesystem in UserSpacE
+import six       # Python 2 and 3 Compatibility Library (2*3=6)
```

Comment:
I don't think these comments are needed, and they are not relevant for supporting python 3.

Also aligning the comments is not compatible with pep8. While we don't enforce pep8 yet, I don't like to introduce incompatible code.

Please check your changes with flake8  to make sure new code conforms to pep8.